File: output/.mvn/wrapper/MavenWrapperDownloaderTest.js
import static org.junit.Assert.assertTrue;

public class MavenWrapperDownloaderTest {

    @Test
    public void testDownloadFile() throws Exception {
        MavenWrapperDownloader.main(new String[] { "." });
        File file = new File(MavenWrapperDownloader.MAVEN_WRAPPER_JAR_PATH);
        assertTrue(file.exists());
    }

File: output/.mvn/wrapper/MavenWrapperDownloaderTest.java
import static org.junit.Assert.assertTrue;

public class MavenWrapperDownloaderTest {

    @Test
    public void testDownloadFileFromURL() throws Exception {
        String url = "https://example.com/maven-wrapper.jar";
        File tempDir = Files.createTempDirectory("maven-wrapper-test").toFile();
        File outputFile = new File(tempDir, "maven-wrapper.jar");
        MavenWrapperDownloader.downloadFileFromURL(url, outputFile);
        assertTrue(outputFile.exists());
    }
